### ISSUE TITLE
Could not build by Carthage and Xcode 10.2b1 ~ b4.

### ISSUE THREAD

**Issue by r-plus** [realm/realm-cocoa#6075]
## Goals
buildable by carthage with Xcode 10.2b1 ~ b4.

## Expected Results
buildable by carthage with Xcode 10.2b1 ~ b4.

## Actual Results
Build failed. 

## Steps to Reproduce
```
$ carthage bootstrap --platform iOS realm-cocoa
*** Checking out realm-cocoa at "v3.13.1"
*** xcodebuild output can be found in /var/folders/hr/43qhs4_16hx73mq7cl47xg4w0000gn/T/carthage-xcodebuild.v5eWl0.log
*** Invalid cache found for realm-cocoa, rebuilding with all downstream dependencies
*** Downloading realm-cocoa.framework binary at "v3.13.1"
***  Skipped installing realm-cocoa.framework binary due to the error:
	"Incompatible Swift version - framework was built with 4.2.1 (swiftlang-1000.11.42 clang-1000.11.45.1) and the local version is 5.0 (swiftlang-1001.0.45.7 clang-1001.0.37.7)."

    Falling back to building from the source
*** Building scheme "Realm" in Realm.xcworkspace
*** Building scheme "RealmSwift" in Realm.xcworkspace
Build Failed
	Task failed with exit code 65:
	/usr/bin/xcrun xcodebuild -workspace /Users/taiki/clone/test/Carthage/Checkouts/realm-cocoa/Carthage/Realm.xcworkspace -scheme RealmSwift -configuration Release -derivedDataPath /Users/taiki/Library/Caches/org.carthage.CarthageKit/DerivedData/10.2_10P82s/realm-cocoa/v3.13.1 -sdk iphoneos ONLY_ACTIVE_ARCH=NO CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES archive -archivePath /var/folders/hr/43qhs4_16hx73mq7cl47xg4w0000gn/T/realm-cocoa SKIP_INSTALL=YES GCC_INSTRUMENT_PROGRAM_FLOW_ARCS=NO CLANG_ENABLE_CODE_COVERAGE=NO STRIP_INSTALLED_PRODUCT=NO (launched in /Users/taiki/clone/test/Carthage/Checkouts/realm-cocoa)

This usually indicates that project itself failed to compile. Please check the xcodebuild log for more details: /var/folders/hr/43qhs4_16hx73mq7cl47xg4w0000gn/T/carthage-xcodebuild.v5eWl0.log
```

in log file, failied in archive phase. 

```
note: Using new build system
note: Planning build
note: Constructing build description
Build system information
error: SWIFT_VERSION '3.0' is unsupported, supported versions are: 4.0, 4.2, 5.0. (in target 'RealmSwift')

Build system information
error: SWIFT_VERSION '3.0' is unsupported, supported versions are: 4.0, 4.2, 5.0. (in target 'RealmSwift')

Build system information
warning: Swift 3 mode has been deprecated and will be removed in a later version of Xcode. Please migrate "RealmSwift" to Swift 4.2 using "Convert > To Current Swift Syntax…" in the Edit menu. (in target 'RealmSwift')

Build system information
warning: Swift 3 mode has been deprecated and will be removed in a later version of Xcode. Please migrate "RealmSwift" to Swift 4.2 using "Convert > To Current Swift Syntax…" in the Edit menu. (in target 'RealmSwift')

** ARCHIVE FAILED **
```

## Version of Realm and Tooling
Realm framework version: 3.13.1
Xcode version: 10.2b1  ~ b4
Dependency manager + version: Carthage 0.32.0

**Comment by allenhumphreys** [comment]
Seems to be fixed in #6076, which you can use via `github "realm/realm-cocoa" "tg/xcode-10.2"` until it is merged

**Comment by stennie** [comment]
Closing this issue with no recent activity. I assume this issue has been resolved since XCode 10.2 GM was released in March, 2019.
